packages/google/openiap/src/horizon/java/dev/hyo/openiap/horizon/OpenIapHorizonModule.kt

```
@@ -0,0 +1,911 @@
+            val currentActivity = activityRef.get() ?: throw Exception("Activity not available")
+
+            val result = suspendCancellableCoroutine<BillingResult> { cont ->
+                try {
+                    val listener = AlternativeBillingOnlyInformationDialogListener { billingResult ->
+                        cont.resume(billingResult)
+                    }
+                    client.showAlternativeBillingOnlyInformationDialog(
+                        currentActivity,
+                        listener
+                    )
+                } catch (e: NoSuchMethodError) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)